doc/sfwbar.rst

```diff
@@ -707,6 +707,30 @@ tricky). Mapping is supported via top-level ``MapAppId`` keyword. I.e. ::
 where app_id is the desired app_id and pattern is a regular expression to
 match the title against.
 
+If you are using an XWayland app, they usually do not have an `app_id` set. If
+an icon is not showing, you can add your icon to the following locations:
+1. `$HOME/.icons`
```

Comment:
sfwbar will also look for icons in:
1) location of the main config file currently in use
2) $XDG_CONFIG_HOME/sfwbar/
3) One of directories listed in $XDG_DATA_DIRS/sfwbar/